src/app/portlets/content-types/fields/content-type-fields-properties-form/field-properties/relationships-property/new-relationships.component.ts/new-relationships.component.ts

```diff
@@ -0,0 +1,98 @@
+import { Component, OnInit, Input, Output, EventEmitter } from '@angular/core';
+import { DotMessageService } from '@services/dot-messages-service';
+import { PaginatorService } from '@services/paginator';
+import { ContentType } from '@portlets/content-types/shared/content-type.model';
+import { RelationshipCardinality } from '@portlets/content-types/fields/shared/relationship-cardinality.model';
+import { RelationshipService } from '@portlets/content-types/fields/service/relationship.service';
+import { DotContentletService } from '@services/dot-contentlet/dot-contentlet.service';
+import { Observable } from 'rxjs';
+
+@Component({
+    providers: [PaginatorService],
+    selector: 'dot-new-relationships',
+    templateUrl: './new-relationships.component.html'
+})
+export class NewRelationshipsComponent implements OnInit {
+    @Input()
+    cardinalityIndex: number;
+
+    @Input()
+    velocityVar: string;
+
+    @Input()
+    editing: boolean;
+
```

Comment:
If this 3 inputs are coming always and change always at the same time, you might want to use an object (if makes sense).